.editorconfig

```diff
@@ -0,0 +1,4 @@
+root = true
```

Comment:
what is this file? is it used go fmt? if not can it be nuked?

Reply:
https://editorconfig.org/

it's not only respect by fmt - it tell all IDEs how to format things

Reply:
Nice.

Won't apply to my crazy setup (translated: I do most dev on a 12 year old Thinkpad running a version of Ubuntu so old it's crazy and I use vanilla vim), but if it works for everyone else then I'm happy.

Reply:
@buro9 well who would say vim is not supported :rocket: 

https://github.com/editorconfig/editorconfig-vim#readme

Reply:
I saw that. But if I were the type of person who installed things to make their life easier I'd have installed an IDE 😁